stack.yaml

```diff
@@ -38,10 +41,3 @@ extra-deps:
   - primitive-unlifted-0.1.3.0
   - run-st-0.1.1.0
   - tuples-0.1.0.0
-
-nix:
-  enable: true
-  packages:
-    - lzma
-    - postgresql
-    - zlib
```

Comment:
Removing in favour of `shell.nix`. Also means that that you don't need to `--no-nix` anymore 👍